src/DependencyInjection/Loader/ConfigurableCallValuesCollectingPhpFileLoader.php

```diff
@@ -24,7 +24,7 @@ public function __construct(
      * @param mixed $resource
      * @param null|string $type
      */
-    public function load($resource, $type = null): void
+    public function load($resource, string $type = null): void
```

Comment:
this update is required as new symfony/dependency-injection release which update the param type